docs/mindsdb-docs/docs/sql/api/retrain.md

```diff
@@ -17,7 +20,19 @@ On execution, we get:
 Query OK, 0 rows affected (0.058 sec)
 ```
 
-## Validation
+## When to `#!sql RETRAIN` the Model?
+
+To find out whether you need to retrain your model, query the `mindsdb.predictors` table and look for the `update_status` column.
```

Comment:
I think we need mention here why `update_status` may become `available`:
that may happen only if update mindsdb version, and previous predictors becomes incompatible with new version.
In general, there is only 2 reasons to retrain predictor:
1. new mindsdb version. Predictor may be still compatible, but because it is new version, it may give better result if retrain
2. if source data is changed
3. (in future) if user want to use another parameters in `USING`

Reply:
Added.